kadi/update_cmds.py

```diff
@@ -399,7 +402,7 @@ def main(args=None):
 
     if pars_dict.n_updated > 0:
         with open(pars_dict_path, 'wb') as fh:
-            pickle.dump(pars_dict, fh, protocol=-1)
+            pickle.dump(pars_dict, fh, protocol=2)
```

Comment:
For our Py2, was protocol=-1 (highest allowed, right?) equivalent to protocol=2?

Reply:
Correct.  For the record:
https://docs.python.org/2/library/pickle.html#data-stream-format

Reply:
Yes, that's what I figured.  So protocol=2 is a no-op for this if run on Py2 and locks it at a Py2-compatible protocol if run on Py3.